src/inverseRobot.js

```diff
@@ -4,10 +4,26 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const arrayItems = [];
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (arrayItems.includes(value)) {
+      return null;
+    }
+
+    if (result[value]) {
+      return null;
+    }
+
+    arrayItems.push(value);
```

Comment:
Do it without using any additional array. use one of checklist [2] examples to check if a property exists